src/StateMachine.cpp

```diff
@@ -168,6 +168,8 @@ StateMachine::Update StateMachine::processSDOReceive(canbus::Message const& msg)
             cmd.size = 4;
```

Comment:
With the latest change, this should move inside the if part

And now that there is an else, invert the two conditions to remove the negation.